packages/sdks/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -513,14 +513,26 @@ class DescopeWc extends BaseDescopeWc {
     if (action === RESPONSE_ACTIONS.poll) {
       // schedule next polling request for 2 seconds from now
       this.#pollingTimeout = setTimeout(async () => {
-        const sdkResp = await this.sdk.flow.next(
-          executionId,
-          stepId,
-          CUSTOM_INTERACTIONS.polling,
-          flowVersion,
-          componentsVersion,
-          {},
-        );
+        console.log('calling next from polling');
+        let sdkResp;
+        try {
+          sdkResp = await this.sdk.flow.next(
+            executionId,
+            stepId,
+            CUSTOM_INTERACTIONS.polling,
+            flowVersion,
+            componentsVersion,
+            {},
+          );
+        } catch (e) {
+          this.#handlePollingResponse(
```

Comment:
This might break compatibility in case someone expected the exeption throw?

Reply:
@dorsha was testing with @itaihanski and @nirgur :)

Please ignore.